roaring/roaring_internal_test.go

```diff
@@ -1542,8 +1547,8 @@ func TestDifferenceBitmapRun(t *testing.T) {
 }
 
 func TestDifferenceBitmapArray(t *testing.T) {
-	a := &container{bitmap: make([]uint64, bitmapN)}
-	b := &container{}
+	a := &container{bitmap: make([]uint64, bitmapN), container_type: ContainerBitmap}
```

Comment:
This test could use more cases - also the bitmap should use a helper function to construct a full bitmap, and we should make sure we're testing the edges of the container. The helper functions bitmapEvens and bitmapOdds are good for this.

Reply:
there is also a MakeBitmap(..)  that will alloc full bitmap with passed in initialize values